libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -283,7 +294,7 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
  if you don't know what kind of change was made to this object and this method
  will try to determine that.
  */
-- (void)userChanged:(SFUserAccountChange)change;
+- (void)userChanged:(SFUserAccount *)user change:(SFUserAccountChange)change;
```

Comment:
You should update the headerdoc for this method to reflect the new property added.